packages/common/src/errors/parse/handlers/zod-parse-error-handler.ts

```diff
@@ -0,0 +1,24 @@
+import { ZodError } from "zod";
+import { fromError } from "zod-validation-error";
+
+import { logger } from "@homarr/log";
+
+import { ParseError } from "../parse-error";
+import { ParseErrorHandler } from "./parse-error-handler";
+
+export class ZodParseErrorHandler extends ParseErrorHandler {
+  handleParseError(error: unknown): ParseError | undefined {
+    if (!(error instanceof ZodError)) return undefined;
+
+    // TODO: migrate to zod v4 prettfyError once it's released
```

Comment:
Can we make a ticket for this?

Reply:
Created https://github.com/homarr-labs/homarr/issues/3122